tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -10,13 +10,11 @@ Test Setup          fail test fast if required
 
 Force Tags          Admin    Local    Dev    AltersData
 
-
 *** Variables ***
```

Comment:
Just wondering if the linting removed these lines?

Reply:
I think this was the robot checker thing, not me, but I'll readd it and see what happens

Reply:
Just double checked - it is the robot linter

Reply:
hmm, can you try doing a pipenv install & touching this file with some empty lines to see if it corrects itself & does the right formatting please?

Reply:
`pipenv install` seems to have sorted it - ta